<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <stylestrategy>PreferDefault</stylestrategy>
   </font>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhFormattedNumbersOnly</set>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>286</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item row="0" column="4">
      <widget class="QLineEdit" name="refRange_3"/>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="refRange"/>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="corrAngle_2"/>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="corrText3">
       <property name="text">
        <string>Δꝺ</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="corrDist_2"/>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="corrAngle"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="corrText1">
       <property name="text">
        <string>Дальность, км</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="corrDist"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="corrText2">
       <property name="text">
        <string>ΔD</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="corrDist_3"/>
     </item>
     <item row="2" column="4">
      <widget class="QLineEdit" name="corrAngle_3"/>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="refRange_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLineEdit" name="alphaOH"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="alphaTarget"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="distCmdrText">
       <property name="text">
        <string>Dк</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="distCmdr"/>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="epsTargetText">
       <property name="text">
        <string>εц</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLineEdit" name="epsTarget"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="alphaTargetText">
       <property name="text">
        <string>αц</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="alphaOHtext">
       <property name="text">
        <string>α OH</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="title_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>-10</y>
      <width>379</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>Поправки</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>671</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="4" column="4">
      <widget class="QLabel" name="ResultPStext">
       <property name="text">
        <string>ПС</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="ResultLVLtext">
       <property name="text">
        <string>Ур</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="ResultDTTtext">
       <property name="text">
        <string>Dцт</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="ResultLVL">
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="ResultDDtext">
       <property name="text">
        <string>ΔD</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="ResultAIMtext">
       <property name="text">
        <string>Прицел</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="ResultRTT">
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLineEdit" name="ResultAIM">
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QLabel" name="ResultdX1kText">
       <property name="text">
        <string>ΔXтыс</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLineEdit" name="ResultShy">
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="ResultRTTtext">
       <property name="text">
        <string>ꝺцт</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QLineEdit" name="ResultPS">
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="ResultDTT">
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="ResultDTR">
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="ResultDTRtext">
       <property name="text">
        <string>Dци</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="ResultShyText">
       <property name="text">
        <string>Шу</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="ResultDRtext">
       <property name="text">
        <string>Δꝺ</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="ResultDD">
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QLineEdit" name="ResultRTR">
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="ResultKyText">
       <property name="text">
        <string>Ку</string>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QLineEdit" name="ResultdX1k">
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLabel" name="ResultRTRtext">
       <property name="text">
        <string>ꝺци</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLineEdit" name="ResultDR">
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="ResultKy">
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="title_3">
       <property name="text">
        <string>Результат</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>604</x>
      <y>180</y>
      <width>101</width>
      <height>50</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="title_4">
       <property name="text">
        <string>Заряд</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="charge">
       <property name="maximum">
        <number>4</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>44</x>
      <y>22</y>
      <width>371</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="1" column="0">
      <widget class="QLabel" name="xOPtext_3">
       <property name="text">
        <string>КНП:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="OPtext">
       <property name="text">
        <string>ОП: </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="xKNPtext">
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="xKNPin"/>
       </item>
       <item>
        <widget class="QLabel" name="yKNPtext">
         <property name="text">
          <string>Y:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="yKNPin"/>
       </item>
       <item>
        <widget class="QLabel" name="hKNPtext">
         <property name="text">
          <string>h:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="hKNPin"/>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="xOPtext">
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="xOPin">
         <property name="tabletTracking">
          <bool>false</bool>
         </property>
         <property name="focusPolicy">
          <enum>Qt::WheelFocus</enum>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhFormattedNumbersOnly</set>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="yOPtext">
         <property name="text">
          <string>Y:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="yOPin"/>
       </item>
       <item>
        <widget class="QLabel" name="hOPtext">
         <property name="text">
          <string>h:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="hOPin"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>171</width>
      <height>13</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>13</height>
     </size>
    </property>
    <property name="text">
     <string>Начальные данные</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>xOPin</tabstop>
  <tabstop>yOPin</tabstop>
  <tabstop>hOPin</tabstop>
  <tabstop>xKNPin</tabstop>
  <tabstop>yKNPin</tabstop>
  <tabstop>hKNPin</tabstop>
  <tabstop>alphaOH</tabstop>
  <tabstop>alphaTarget</tabstop>
  <tabstop>distCmdr</tabstop>
  <tabstop>epsTarget</tabstop>
  <tabstop>refRange</tabstop>
  <tabstop>refRange_2</tabstop>
  <tabstop>refRange_3</tabstop>
  <tabstop>corrDist</tabstop>
  <tabstop>corrDist_2</tabstop>
  <tabstop>corrDist_3</tabstop>
  <tabstop>corrAngle</tabstop>
  <tabstop>corrAngle_2</tabstop>
  <tabstop>corrAngle_3</tabstop>
  <tabstop>charge</tabstop>
  <tabstop>ResultDTT</tabstop>
  <tabstop>ResultDD</tabstop>
  <tabstop>ResultDTR</tabstop>
  <tabstop>ResultRTT</tabstop>
  <tabstop>ResultDR</tabstop>
  <tabstop>ResultRTR</tabstop>
  <tabstop>ResultAIM</tabstop>
  <tabstop>ResultLVL</tabstop>
  <tabstop>ResultKy</tabstop>
  <tabstop>ResultShy</tabstop>
  <tabstop>ResultPS</tabstop>
  <tabstop>ResultdX1k</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
